### ISSUE TITLE
How to include sets of files, and exclude sets

### ISSUE THREAD

**Issue by johnpapa** [es-analysis/plato#111]
I'd like to include a set of files and then exclude a set. Specifically include src/client/**/*.js and exclude src/client/**/*.spec.js and src/client/test 

How can this be done?

**Comment by philstrong** [comment]
Yeah also ran into this "problem" when utilizing plato w/ gulp (re: node), would be nice if you could pass a glob that excluded as well.  For some reason it treated my !foo/bar/*.js as a 2nd inclusion; expect exclusion... bug?

e.g.

    plato.inspect(['foo/**/*.js', '!foo/bar/*.js'], {}...

using options.exclude /foo\/bar\/*/ did the trick

**Comment by michahell** [comment]
@philstrong could you please share how you did this? I'm getting errors whenever i'm trying to set  exclude to anything inside the options object:
```javascript
var options = {
    title: 'some title',
    exclude: '../../../some-url/some-file.js'
  };
```

**Comment by wsain001** [comment]
**Workaround**

`npm install --save-dev glob-all`

`var glob = require('glob-all');`

In your gulp task:

```
var files = glob.sync([
    './src/**/*.js',     // include
    '!./src/**/*spec.js' // exclude
    ]);

plato.inspect(files, your_output_dir, your_options, your_callback);
```

**Comment by philstrong** [comment]
Well played wsain001

**Comment by michahell** [comment]
Nice. any news on ES compat ?
